twitter/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'java'
+}
+
+version 'unspecified'
```

Comment:
why the version is unspecified ? how about 0.1-SNAPSHOT ?

Reply:
I added it to the gradle.properties file.